airflow/www/views.py

```diff
@@ -2187,9 +2188,33 @@ def get_int_arg(value, default=0):
             paging=wwwutils.generate_pages(current_page, num_of_pages,
                                            search=arg_search_query,
                                            showPaused=not hide_paused),
+            now=dt.datetime.now,
+            airflow_timezone=settings.TIMEZONE,
+            client_timezone=CLIENT_TIMEZONE,
+            stringify_timedelta=stringify_timedelta,
             auto_complete_data=auto_complete_data)
 
 
+def stringify_timedelta(delta: dt.timedelta) -> str:
+    seconds = delta.total_seconds()
+    sign_word = ' ago' if seconds < 0 else ' from now'
+    seconds = abs(int(seconds))
+
+    t = 7 * 24 * 60 * 60
+    weeks, seconds = divmod(seconds, t)
+    t /= 7
+    days, seconds = divmod(seconds, t)
+    t /= 24
+    hours, seconds = divmod(seconds, t)
+    t /= 60
+    minutes, seconds = divmod(seconds, t)
+
+    display = [(weeks, 'w'), (days, 'd'), (hours, 'h'), (minutes, 'm'), (seconds, 's')]
+    display = itertools.dropwhile(lambda t: not t[0], display)
+    words = map(lambda t: '%d%s' % t, display)
+    return ' '.join(words) + sign_word if words else 'now'
```

Comment:
If you do my previous suggestion of changing `sign_word` then you don't need the if check here:
```suggestion
    return ' '.join(words) + sign_word
```